lib/scss/inc/_variables.scss

```diff
@@ -12,16 +12,15 @@ $screen-md-max: ($screen-lg-min - 1);
 $border-color: #f0f0f0;
 $border-inverse-color: #2f2f2f;
 $muted-bg: #f9f9f9;
-$text-muted-color: #606060;
+$text-muted-color: #777;
 
 // Colors
 $black: #000;
 $white: #fff;
 
-
 // Body
 $body-bg: $white;
-$body-color: $black;
+$body-color: #707070;
```

Comment:
I think we also wanted to keep the body color as black to make the text more readable as per AB's feedback